plugins/scanners/fossid/src/test/kotlin/FossIdNamingProviderTest.kt

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2021 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.scanners.fossid
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.equals.shouldBeEqual
+import io.kotest.matchers.ints.shouldBeLessThanOrEqual
+
+import io.mockk.every
+import io.mockk.mockkStatic
+
+import java.time.LocalDateTime
+
+class FossIdNamingProviderTest : WordSpec({
+    "createScanCode" should {
+        val namingProvider = FossIdNamingProvider(null, null, emptyMap())
+
+        val mockedDateTime = LocalDateTime.of(2024, 4, 1, 10, 0)
+        val expectedTimestamp = "20240401_100000"
+
+        "create code without branch name, when it's empty" {
+            mockkStatic(LocalDateTime::class) {
```

Comment:
When using `mockStatic`, it is required to reset mocking afterward to prevent that other tests are affected. This is easily achieved by adding an `afterSpec` method that contains an `unmockAll()` call.

Reply:
Probably using `afterEach` is yet safer, to prevent unintended side-effects between individual test cases.